src/Component.elm

```diff
@@ -1,9 +1,32 @@
-module Component exposing (materialIcon)
+module Component exposing (empty, materialIcon, progressBar)
```

Comment:
One reason people in Elm are avoiding using `Component` term is because it is quite overloaded with multiple frameworks. Also especially new comers tend to expect Elm modules to be same as React components, which they are not.

Of course in your own project you know everything, so naming isn't an issue. Just wanted to share some reasoning I know :)